### DISCUSSION TITLE
unable to work with Google Domains

### DISCUSSION THREAD

**Discussion by philross88** [projectdiscovery/interactsh#132] [Q&A] [answered]
I have setup Google domain for hosted server setup. Following the instructions, I set ns1.domiain.tld as ns recording pointing to my VPS. Similarly ns2.domains.tld was also setup as ns record pointing to VPS IP. 

Now I can't change the Nameserver in Google Domains to ns1.domains.tld -- it just fails. Running with debug shows the DNS validation is failing with letsencrypt so I am unable to setup my own instance.

Anyhelp would be great here. Google is not allowing to setup customer DNS when I setup NS records in the register with the same name. For e.g. if I have n1.domain.tld ns record pointing to VPS, I can set ns.1.domain.tld as custom name server.

**Comment by ehsandeep** [comment, marked as answer]
Hey @philross88, Thank you for bringing this up; I'll try to set up an interactsh server with a Google Domains domain.

For the time being, I can confirm two providers, GoDaddy and Namecheap, that are suitable with interactsh domain setup.
